okhttp/src/main/java/io/grpc/transport/okhttp/OkHttpClientTransport.java

```diff
@@ -316,7 +316,7 @@ private boolean startPendingStreams() {
   }
 
   @Override
-  public void start(Listener listener) {
+  public void start(final Listener listener) {
```

Comment:
Even if this is not final I think it should work, because `listener` is a field that is exactly the same as this local variable.